crates/rpc-types-trace/src/parity.rs

```diff
@@ -821,4 +822,31 @@ mod tests {
         let serialized = serde_json::to_string_pretty(&trace).unwrap();
         similar_asserts::assert_eq!(serialized, reference_data);
     }
+    #[test]
+    fn test_nethermind_trace_result_null_output_value() {
+        let reference_data = r#"{
+  "output": null,
```

Comment:
is there a nethermind issue for this, I want to understand when this can be null to rule out something we may have missed

Reply:
To be honest, I didn't dig very deeply, but it seems that vm_trace can also be null.